cmd/qri.go

```diff
@@ -32,6 +32,10 @@ https://github.com/qri-io/qri/issues`,
 			if err == nil && noColor {
 				setNoColor(noColor)
 			}
+			// if it's a tty terminal disable color
+			if ioStreams.IsTerminal() {
+				setNoColor(noColor)
```

Comment:
What happens when the user supplies `qri --no-color=false`?

Reply:
Good point, though I don't see how both of it should work at the same time. I guess leaving the option to the user makes sense. Though I will check if the flag was changed and not just default-ed.